test/common/stats/stat_test_utility.cc

```diff
@@ -83,12 +83,12 @@ void forEachSampleStat(int num_clusters, std::function<void(absl::string_view)>
 
   // These are the other stats that appear in the admin /stats request when made
   // prior to any requests.
-  static const char* other_stats[] = {"http.admin.downstream_cx_length_ms",
```

Comment:
Consider adding an exclusion for creating histograms ending with "_ms" in check_format.py, so that none of these sneak in later.

Reply:
Great idea, added.